index.html

```diff
@@ -3096,12 +3095,12 @@ <h6 data-lang="zh-hant">發表對本文檔的建議</h6>
       <p data-lang="zh-hant">同行標題主要用於小標，以下介紹幾個同行標題的配置範例。同行標題與之後接續內文的空白量，一般為版心設定文字尺寸的全形空白。此外，同行標題可以配置於頁末。</p>
       <ol>
         <li>
-          <p data-lang="en"> The run-in heading is set with the same character size as the main text and with Hei or Song. </p>
+          <p data-lang="en">The run-in heading is set with the same character size as the main text and in Hei or Song. </p>
           <p data-lang="zh-hans" class="checkme">与内文采相同文字尺寸，字体改为黑体或者楷体。</p>
           <p data-lang="zh-hant">與內文採相同文字尺寸，字體改為黑體或者楷體。</p>
         </li>
         <li>
-          <p data-lang="en">Set the run-in heading with one level smaller character size than the main text and use Hei or Song.</p>
+          <p data-lang="en">Set the run-in heading in a character size not smaller than the main text and use Hei or Song.</p>
```

Comment:
The meaning has changed here - presumably it now follows the original chinese text better?

Reply:
Yes, it now follows the original Chinese text better. But I found another issue when reading this sentence: "Song" should actually be "Kai" here, according to the original Chinese text.

Reply:
Will make the appropriate change